Worksheet-B/README.MD

```diff
@@ -0,0 +1,26 @@
+Pseudocode:
+```
+Generate 10 words which are 5 letters long
+Select one word at random as the answer, this word = chosenword
+Display the 10 words
+
+sequence starts here
+
+ If  
+  userinput = chosenword
+  unlock terminal
+  end if
+  
+ elseif 
+  the user has <3 strikes 
+  tell user number of mutual letters of userinput and chosenword
+  add a strike and repeat sequence
+
+ elseif the user has >3 strikes 
+  lock terminal and close terminal.
+  Player loses.
+  endif
+sequence ends here
+```
+Flowchart:
+                       ![flowchart](terminal game flowchart.png)
```

Comment:
Flowchart looks good 👍 

Minor nitpick: you should specify that the number of strikes starts at 0.